### PR TITLE
[QACI-725] [QACI-724] - Update glassfish properties and README through maven install

### PR DESCRIPTION
**Pull request by cubastanley**
## Description
This PR adds support to the ant build for updating major_version minor_version and update_version values in the glassfish properties file with values from the maven pom properties.
It also adds support for generating these values from the version number so as to avoid potential for user error when performing the replacement

### Blockers
N/A

### New tests
N/A

### Testing Performed
Software has been build and run using a handful of different test versions

### Testing Environment
Ubuntu 20.04 in WSL2 - openjdk 1.8.0_322 - Maven 3.8.5

## Documentation
N/A

## Notes for Reviewers
Source for the plugin that performs the version number split is viewable here: https://github.com/cubastanley/ecosystem-dynamic-build-properties-maven-plugin/tree/v0.0.1.a2

### CODE REVIEW

**Comment by cubastanley** [comment]
Jenkins test please

File: pom.xml [hunk 1]
```diff
@@ -380,8 +393,32 @@
     <build>
         <defaultGoal>install</defaultGoal>
 
+        <plugins>
+            <plugin>
+                <groupId>fish.payara.maven.plugins</groupId>
+                <artifactId>dynamic-build-properties-maven-plugin</artifactId>
+                <version>0.0.1.a2</version>
```
Comment (Pandrex247) [inline pom.xml:400]: We use properties

File: appserver/packager/appserver-base/pom.xml [hunk 2]
```diff
@@ -99,6 +99,35 @@
                     </execution>
                 </executions>
             </plugin>
+             <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>3.1.0</version>
```
Comment (Pandrex247) [inline appserver/packager/appserver-base/pom.xml:104]: Already defined in pluginManagement

**Comment by cubastanley** [comment]
Jenkins test please

**Comment by cubastanley** [comment]
Jenkins test please

**Comment by cubastanley** [comment]
Jenkins test please

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**